run_scripts/run_pipeline.sh

```diff
@@ -22,9 +22,9 @@ DATA_BACKUPS_DIR=$8
 
 ./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
 
-./2_fetch_raw_data.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
+./2_fetch_raw_data.sh "$USER" "$PIPELINE_CONFIGURATION_FILE_PATH" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$DATA_ROOT"
 
-./3_generate_outputs.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
+./3_generate_outputs.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$DATA_ROOT"
```

Comment:
Surely you still need the pipeline configuration file path here?

Reply:
So the pipeline copies the pipeline configuration file as a DockerFile step here https://github.com/AfricasVoices/Project-ICRAF/blob/34873923c4c1091a46d170f7ad5267de3c7d7180/Dockerfile#L33 and passes the path in the docker-run.sh here https://github.com/AfricasVoices/Project-ICRAF/blob/34873923c4c1091a46d170f7ad5267de3c7d7180/docker-run.sh#L60. Therefore the argument was never needed in this script. My thinking here is to remove this and standardise this as is on other pipelines but in a different PR. @as2388  what your thoughts?

Reply:
Ah, I see. Yes, the config should be moved out of the Dockerfile and into an argument, but not in this PR.

Reply:
Great merging!